configure.ac

```diff
@@ -76,6 +76,24 @@ AC_ARG_ENABLE(nss,
 AC_ARG_ENABLE(pic,
    AS_HELP_STRING(--enable-pic, [Build Tor's binaries as position-independent code, suitable to link as a library.]))
 
+AC_ARG_ENABLE(missing-doc-warnings,
+   AS_HELP_STRING(--enable-missing-doc-warnings, [Tell doxygen to warn about missing documentation.]))
```

Comment:
This seems like it would make all Doxygen warnings (not just missing documentation) non-fatal, which seems like a reasonable constraint. Maybe we want to highlight this side effect in the help text and changes file?